source/mainnet/net/guides/developer-page.rst

```diff
@@ -28,6 +28,8 @@ Official resources
 
 - See :ref:`dApp examples<dapp-examples>` for a list of dApp examples and their resources.
 
+- For testing purposes, an Ubuntu node is available on testnet to use when testing smart contracts and dApps. You can use this node for API calls of chain methods only with GRPC v1, gRPC v2, and gRPC web. The address is node.testnet.concordium.com on port 10000 (GRPCv1) and port 20000 (gRPCv2 and gRPC-web). This node is periodically maintained by Concordium. The status of this node is available on the `Testnet status page <https://status.testnet.concordium.software>`__.
```

Comment:
Why does it matter that it is an Ubuntu node? I recommend removing this. What is important is that it has grpc... exposed on ports 10000 and 20000 as is stated.